README.md

```diff
@@ -157,15 +155,15 @@ $ systemctl start snap-telemetry
 If you installed Snap from binary, you can start Snap daemon via the command:
 ```
 $ sudo mkdir -p /var/log/snap
-$ sudo snapd --plugin-trust 0 --log-level 1 -o /var/log/snap &
+$ sudo snapteld --plugin-trust 0 --log-level 1 -o /var/log/snap &
```

Comment:
Should we change the `-o` to `--log-path` to match the other self-explanatory options?